File: level2_sacco.py
# Level 2 - SACCO Member Savings (Python)

# Member details
name = input("Enter member name: ")
member_id = input("Enter member ID: ")

total_savings = 0

# Input contributions for 6 months
for month in range(1, 7):
    contribution = input(f"Enter contribution for month {month}: ")
    # Convert to float (type coercion)
    contribution = float(contribution)
    total_savings += contribution

# Print results
print("\nMember Name:", name)
print("Member ID:", member_id)
print("Total Savings:", total_savings)

File: level2_sacco.cpp
#include <iostream>
using namespace std;

int main() {
    string name, member_id;
    float contribution, total_savings = 0;

    cout << "Enter member name: ";
    cin >> name;

    cout << "Enter member ID: ";
    cin >> member_id;

    for (int month = 1; month <= 6; month++) {
        cout << "Enter contribution for month " << month << ": ";
        cin >> contribution;
        total_savings += contribution;
    }

    cout << "\nMember Name: " << name << endl;
    cout << "Member ID: " << member_id << endl;
    cout << "Total Savings: " << total_savings << endl;

    return 0;
}